cmd/kubectl-directpv/label_volumes.go

```diff
@@ -0,0 +1,180 @@
+// This file is part of MinIO DirectPV
+// Copyright (c) 2021, 2022 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/minio/directpv/pkg/apis/directpv.min.io/types"
+	"github.com/minio/directpv/pkg/client"
+	"github.com/minio/directpv/pkg/consts"
+	"github.com/minio/directpv/pkg/utils"
+	"github.com/minio/directpv/pkg/volume"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var labelVolumesCmd = &cobra.Command{
+	Use:           "volumes k=v|k-",
+	Aliases:       []string{"volume", "vol"},
+	Short:         "Set labels to volumes",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Example: strings.ReplaceAll(
+		`# Set 'tier: hot' label to all volumes in all nodes
+$ kubectl {PLUGIN_NAME} label volumes tier=hot --all
+
+# Set 'type: fast' to volumes allocated in specific drives from a node
+$ kubectl {PLUGIN_NAME} label volumes type=fast --nodes=node1 --drives=nvme1n{1...3}
+
+# Remove  'tier: hot' label from all volumes in all nodes
+$ kubectl {PLUGIN_NAME} label volumes tier- --all`,
+		`{PLUGIN_NAME}`,
+		consts.AppName,
+	),
+	Run: func(c *cobra.Command, args []string) {
+		volumeNameArgs = idArgs
+		if err := validateListVolumesArgs(); err != nil {
```

Comment:
You would need to have validations in separate function `validateLabelVolumesCmd()`